library/src/main/java/com/novoda/downloadmanager/LiteDownloadBatchStatus.java

```diff
@@ -81,6 +82,7 @@ public long downloadedDateTimeInMillis() {
         return downloadedDateTimeInMillis;
     }
 
+    @WorkerThread
```

Comment:
I thought `DOWNLOADING` came from the `download-manager` and therefore was already on an executor, not on the main thread.